docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -178,38 +188,39 @@ return (
 
 {{"demo": "RowGroupingUseKeepGroupedColumnsHidden.js", "bg": "inline", "defaultCodeOpen": false}}
 
-## Disable the row grouping
+## Disable row grouping
 
 ### For all columns
 
-You can disable row grouping by setting the `disableRowGrouping` prop to true.
-
-It will disable all the features related to the row grouping, even if a model is provided.
+To disable row grouping for all columns, set the `disableRowGrouping` prop to `true`.
+This disables all features related to row grouping, even if a model is provided.
 
 {{"demo": "RowGroupingDisabled.js", "bg": "inline", "defaultCodeOpen": false}}
 
-### For some columns
+### For specific columns
+
+To disable grouping for a specific column, set the `groupable` property on its `GridColDef` to `false`.
 
-In case you need to disable grouping on specific column(s), set the `groupable` property on the respective column definition (`GridColDef`) to `false`.
-In the example below, the `director` column cannot be grouped. In all examples, the `title` and `gross` columns cannot be grouped.
+In the example below, the **Director** column cannot be grouped, even though there are repeating values.
+(The **Title** and **Gross** columns cannot be grouped in any examples in this doc because there are no repeating values.)
 
 {{"demo": "RowGroupingColDefCanBeGrouped.js", "bg": "inline", "defaultCodeOpen": false}}
 
-### Grouping non-groupable columns programmatically
+### Grouping non-groupable columns
 
-To apply row grouping programmatically on non-groupable columns (columns with `groupable: false` in the [column definition](/x/api/data-grid/grid-col-def/)), you can provide row grouping model in one of the following ways:
+To apply row grouping programmatically on non-groupable columns (columns with `groupable: false` in the `GridColDef`), you can provide the row grouping model in one of three ways:
```

Comment:
I find having a link to the reference API helpful. Just curious if the rationale for removing it is to avoid potential misdirection?

Reply:
It seemed redundant and maybe a little arbitrary to link to it here since the column definition is mentioned but not linked in other places throughout the doc. If it's specifically useful in this context, then maybe we should be more explicit about calling it out as a resource here. Since this is immediately followed by a list, how about an info or success callout below that directing the reader to the API reference for more complete details?